draft-dnsop-deleg.md

```diff
@@ -149,60 +147,23 @@ NS based delegation supports DNS over UDP and TCP, but does not have the ability
 * Support for Abstract Delegation: the SVCB record format has support for an Alias Form record. In the case of DELEG, the AliasForm record enables a domain owner to indicate that their zone will be hosted elsewhere, like a DNS service provider, in a way that enables the service provider to update their authoritative information without coordination with the domain owner, domain registrar. 
 * Authenticated and Parent Centric: While NS records are authoritative at the child, some resolver algorithms are parent centric while others are child centric. With DELEG, this ambiguity is removed and parent centricity and authority is specified. This decision also enables the records to be signed in the parent zone, reducing the potential risk of a denial of service for clients when being delegated. Additionally, a parent centric record is required to support resolution with a cold cache and duplication of data in the child zone can lead to inconsistencies as the records change over time.
 * Able to coexist in the ecosystem: DELEG is defined as being a record in the parent, signifying a zone cut. As a result of that design decision, additional effort and time will be required to deploy DELEG to all levels of the DNS hierarchy, especially when considering the Root zone and TLDs. To support the deployment, DELEG must be able to coexist within the ecosystem with the existing NS based methods of resolution. Testing has been done to show that many deployed resolvers can handle DELEG and NS records side-by-side to enable a rollout. 
+* While DELEG can be used in an unsigned zone, it is recommended to use DNSSEC. The DELEG record must be signed or denied in the parent zone when it is signed. Certain security propperties might not be available and hence certain features only will work when the DELEG record is signed
```

Comment:
nit: end the sentence with a period.
s/propperties/properties/

I find the switch from "The DELEG record must be signed or denied in the parent zone when it is signed." to the next sentence makes it hard to follow. I would preface the last sentence with: "Without DNSSEC, certain security properties might...." just to switch back to the context.

Reply:
Addressed this. Thanks a lot.